RELEASE_NOTES.md

```diff
@@ -1,6 +1,16 @@
 Authentication Service MKII release notes
 =========================================
 
+0.3.0
+-----
+* CONFIGURATION CHANGE - the `identity-provider-Google-custom-people-api-host`
```

Comment:
Actually we probably need to add this to the template as well.

Reply:
https://github.com/kbase/auth2/pull/323